user-guide/Advanced_Modules/Swarming/Messages/ElementUnavailableWhileSwarming.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: ElementUnavailableWhileSwarming
+---
+
+# Message: "x" is currently switching
+
+When an element is switching over from being hosted on one DataMiner agent to being hosted on another DataMiner agent, it will be temporarily unavailable for interacting with.
```

Comment:
" for interacting with." sounds a bit strange IMO, but ill let F1 decide

Reply:
tweaked a bit